shared/src/main/scala/FuncHelpers.scala

```diff
@@ -9,52 +9,24 @@ object FuncHelpers:
     val res =
       fn.arity match
         case 1 =>
-          val a = ctx.pop()
-          ListHelpers.makeIterable(a).vmap { a =>
-            Interpreter.executeFn(fn, args = Some(List(a)))
+          ListHelpers.makeIterable(ctx.pop()).vmap { a =>
+            Interpreter.executeFn(fn, args = List(a))
           }
         case 2 =>
           val b, a = ctx.pop()
-          VList(ListHelpers.makeIterable(a).zipWithIndex.map { (a, i) =>
-            Interpreter.executeFn(
-              fn,
-              args = Some(List(a, b))
-            )
-          }*)
+          ListHelpers.makeIterable(a).vmap { a =>
```

Comment:
I gotta say, dyad vectorisation still confuses me. Why isn't `b` also turned into an iterable and zipped with `a`?

Reply:
Because if you wanted that behaviour, you wouldn't need `v` for things that already vectorise. And for things that don't automatically vectorise, zipping is only needed maybe 10% of the time at most.

However, I'm thinking of having another modifier that uses the behaviour of vectorises from before this PR.

Reply:
In another PR that is - it'd have to be discussed for which codepage item it'd take, etc.